### ISSUE TITLE
[ENHANCEMENT] Chart titles are not accessible

### ISSUE THREAD

**Issue by BenGWeeks** [lnbits/lnbits#3035]
**Describe the bug**

Chart text should be contrasting enough to be able to be easily read (needed for accessibility).

**To Reproduce**

1. Go to 'v1.lnbits.com'
2. Login
3. Ensure wallet has historically data
4. Ensure wallet charts are enabled.
5. See wallet charts

**Expected behavior**

Text should be clear and contrast against the background.

**Screenshots**

![Image](https://github.com/user-attachments/assets/b6d228c0-ea9d-4a60-ba84-cd601bbe7f48)

**Desktop (please complete the following information):**

- LNbits version: v1.0.0rc7
